src/cli/backup/exchange_e2e_test.go

```diff
@@ -561,9 +561,10 @@ func runExchangeDetailsCmdTest(suite *PreparedBackupExchangeE2ESuite, category p
 
 type BackupDeleteExchangeE2ESuite struct {
 	tester.Suite
-	dpnd              dependencies
-	backupOp          operations.BackupOperation
-	secondaryBackupOp operations.BackupOperation
+	dpnd           dependencies
+	backupOp       operations.BackupOperation
+	secondBackupOp operations.BackupOperation
+	thirdBackupOp  operations.BackupOperation
```

Comment:
could we replace this with a slice and index, and assign an index to a given test to use?